api_app/api/routes/airlock_resource_helpers.py

```diff
@@ -35,15 +37,16 @@ async def save_and_publish_event_airlock_request(airlock_request: AirlockRequest
     try:
         logging.debug(f"Sending status changed event for airlock request item: {airlock_request.id}")
         await send_status_changed_event(airlock_request)
-        # TODO - replace with the emails list
-        await send_airlock_notification_event(airlock_request, [], [])
+        access_service = get_access_service()
+        role_assignment_details = access_service.get_workspace_role_assignment_details(workspace)
+        await send_airlock_notification_event(airlock_request, role_assignment_details["researcher_emails"], role_assignment_details["owner_emails"])
```

Comment:
@LizaShak @anatbal you didn't define type hints for send_airlock_notification_event, but I suspect you are passing a list? 
(The logic app accepts a `;` delimited list)